src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Microsoft.Extensions.Telemetry.Abstractions.csproj

```
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>Microsoft.Extensions.Telemetry</RootNamespace>
+    <TargetFrameworks>$(NetCoreTargetFrameworks);netstandard2.0;</TargetFrameworks>
```

Actually, this removes net462 tfm. Should it be 
```suggestion
    <TargetFrameworks>$(NetCoreTargetFrameworks);netstandard2.0;net462</TargetFrameworks>
```
instead?

---

@evgenyfedorov2 But that's intentional because net462 supports netstandard2.0, right? https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0